README.md

```diff
@@ -1,23 +1,9 @@
 SimpleLogin Chrome/Firefox extension
 ---
-<p>
```

Comment:
ah this allows to having some nice badges on the readme, why do we need to remove them?

Reply:
This is due to my stupid git command, fixed in the last push